### PR TITLE
[webui][api] Simplify routes by using optional segments

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/test/functional/webui/apidocs_controller_test.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Webui::ApidocsControllerTest < Webui::IntegrationTest
   def test_index
     return unless File.exist? '/var/adm/fillup-templates'
     # test relative links
-    visit apidocs_path
+    visit "/apidocs"
```
Comment (mdeniz) [inline src/api/test/functional/webui/apidocs_controller_test.rb]: Why don't you use a named route to have the `apidocs_path` method still working?
I mean adding `as: 'apidocs'` in the rewriting of the route. Like this:
```
get 'apidocs(/index)', to: 'webui/apidocs#index', as: 'apidocs'
```

Comment (bgeuken) [inline src/api/test/functional/webui/apidocs_controller_test.rb]: I didn't want to define a named route just for one use case. Especially not for a test

Comment (mdeniz) [inline src/api/test/functional/webui/apidocs_controller_test.rb]: Ok :smile:

**Comment by mdeniz** [comment]
So, LGTM :+1:

**Pull request merged**